sdk/sphere/Azure.ResourceManager.Sphere/api/Azure.ResourceManager.Sphere.netstandard2.0.cs

```diff
@@ -0,0 +1,611 @@
+        public static implicit operator Azure.ResourceManager.Sphere.Models.ImageType (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.Sphere.Models.ImageType left, Azure.ResourceManager.Sphere.Models.ImageType right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class ListDeviceGroupsContent
+    {
+        public ListDeviceGroupsContent() { }
+        public string DeviceGroupName { get { throw null; } set { } }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct OSFeedType : System.IEquatable<Azure.ResourceManager.Sphere.Models.OSFeedType>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public OSFeedType(string value) { throw null; }
+        public static Azure.ResourceManager.Sphere.Models.OSFeedType Retail { get { throw null; } }
+        public static Azure.ResourceManager.Sphere.Models.OSFeedType RetailEval { get { throw null; } }
+        public bool Equals(Azure.ResourceManager.Sphere.Models.OSFeedType other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.Sphere.Models.OSFeedType left, Azure.ResourceManager.Sphere.Models.OSFeedType right) { throw null; }
+        public static implicit operator Azure.ResourceManager.Sphere.Models.OSFeedType (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.Sphere.Models.OSFeedType left, Azure.ResourceManager.Sphere.Models.OSFeedType right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class ProductPatch
+    {
+        public ProductPatch() { }
+        public string Description { get { throw null; } set { } }
+    }
+    public partial class ProofOfPossessionNonceContent
+    {
+        public ProofOfPossessionNonceContent(string proofOfPossessionNonce) { }
+        public string ProofOfPossessionNonce { get { throw null; } }
+    }
+    public partial class ProofOfPossessionNonceResponse : Azure.ResourceManager.Sphere.Models.CertificateProperties
+    {
+        internal ProofOfPossessionNonceResponse() { }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct ProvisioningState : System.IEquatable<Azure.ResourceManager.Sphere.Models.ProvisioningState>
```

Comment:
Could we rename this type to `SphereProvisioningState`?

Reply:
fixed in latest commit